assets/js/stripe-payment-request.js

```diff
@@ -609,9 +609,21 @@ jQuery( function( $ ) {
 					paymentRequest.show();
 				}
 			});
+			
+			$( document.body ).on( 'wc_stripe_unblock_payment_request_button wc_stripe_enable_payment_request_button', function () {
```

Comment:
TIL that you cold put multiple events in jQuery event registrations for 1 handler 😮

Reply:
it's been years since last time I've worked with jQuery :D